content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/session-certificates.md

```diff
@@ -0,0 +1,77 @@
+```shell
+export HOLDING_ID=<holding-id>
+```
+<!--add links to two section where holding ID is retrieved?-->
+
+## Generate a Certificate Signing Request (CSR)
+
+After creating the MGM or member session key pair, but before building the registration context, generate a CSR for the session certificate by running the following command, replacing `X500_NAME` with the X500Name of the MGM or member:
+```shell
+curl --fail-with-body -s -S -k -u admin:admin  -X POST -H "Content-Type: application/json" -d '{"x500Name": "'$X500_NAME'"}' $API_URL"/certificates/"$HOLDING_ID/$SESSION_KEY_ID > $WORK_DIR/request.csr
+```
+Similarly to the TLS certificate, the CSR can be processed to issue a certificate using either a real CA of your choice or the fake CA dev tool. To use the fake CA tool:
+```shell
+cd $RUNTIME_OS
+java -jar ./applications/tools/p2p-test/fake-ca/build/bin/corda-fake-ca-5.0.0.0-SNAPSHOT.jar -m /tmp/ca csr $WORK_DIR/request.csr
+cd $WORK_DIR
+```
+This command outputs the location of the signed certificate. For example:
+```shell
+Wrote certificate to /tmp/ca/request1/certificate.pem
+```
+At this point, you should already have a certificate based on the CSR exported from Corda issued either by a real CA or by the fake CA tool. You must upload the certificate chain to the Corda cluster. To upload the certificate chain, run:
+```shell
+curl -k -u admin:admin -X PUT  -F certificate=@/tmp/ca/request/certificate.pem -F alias=session-certificate $API_URL/certificates/vnode/$HOLDING_ID/p2p-session
+```
+You can optionally omit the root certificate.
+
+{{< note >}}
+If you upload a certificate chain consisting of more than one certificates, ensure that `-----END CERTIFICATE-----` and `-----BEGIN CERTIFICATE-----` from the next certificate are separated by a new line with no empty spaces in between.
+{{< /note >}}
+
+### Disable Revocation Checks
+
+If the CA has not been configured with revocation (for example, via CRL or OCSP), you can disable revocation checks. By default, revocation checks are enabled.
+The fake CA dev tool does not support revocation and so, if you are using the fake CA, you must disable revocation checks.
+To disable revocation checks, do the following:
+1. Retrieve the current link manager configuration version:
+   ```shell
+   curl --insecure -u admin:admin -X GET $API_URL/config/corda.p2p.linkManager
+   ```
+2. Save the displayed version number from the response as a variable:
+   ```shell
+   export CONFIG_VERSION=<configuration-version>
+   ```
+3. Send the following request to disable revocation checks for the specified gateway worker:
+   ```
+   curl -k -u admin:admin -X PUT -d '{"section":"corda.p2p.linkManager", "version":"'$CONFIG_VERSION'", "config": "{ "revocationCheck": { "mode": "OFF" } }", "schemaVersion": {"major": 1, "minor": 0}}' $API_URL"/config"
+   ```
+
+## Build Registration Context for MGM Registration
```

Comment:
I would move this section to the top after `## Set Variables`.
This section is in regards to the MGM onboarding, whereas the rest of the doc is in regards to member onboarding. 
It think we should link to this from the "build registration context" of the MGM onboarding doc also.